File: src/javascript/number_triangle.js
function GenerateTriangle(Number) {
    //Variable
    var pyramid;
    var i;
    let results = '';

    //Making piramid Use foreach
    for (i = Number; i >= 1; i--) {
        for (pyramid = 1; pyramid <= i; pyramid++) {
            results += pyramid + ' ';
        }
        console.log(results);
        results = '';
    }
}
//Call The function
GenerateTriangle(5)


/**
Output:

1 2 3 4 5 
1 2 3 4 
1 2 3 
1 2 
1 
*/

File: src/c/number_triangle.c
#include <stdio.h>

int main() {
   //Variable
   int i, triangle, number;

   //Inputing Number Of the rows
   printf("Enter the number: ");
   scanf("%d", &number);

   //Making pyramid triangle number with foreach
    for (i = number; i >= 1; --i) {
        for (triangle = 1; triangle <= i; ++triangle) {
            printf("%d ", triangle);
        }
        printf("\n");
    }
   return 0;
}
